src/jaegertracing/LogRecord.h

```diff
@@ -17,6 +17,8 @@
 #ifndef JAEGERTRACING_LOGRECORD_H
 #define JAEGERTRACING_LOGRECORD_H
 
+#include "jaegertracing/Compilers.h"
```

Comment:
~~As I've written before, this should be unnecessary. For instance, `Tag.h` should already include `Config.h`, which includes `Compilers.h`. Rule of thumb: you don't need to include a header already included in `Config.h`.~~

I'm sorry to say I got confused between this config header and one in the C client I have been working on (https://github.com/isaachier/jaeger-client-c/blob/master/src/jaegertracingc/config.h.in). My mistake. Keep this include wherever you need it.